src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -0,0 +1,512 @@
+                id="dayOfWeek"
+                name="dayOfWeek"
+                disabled={!(frequency === 'weekly')}
+                options={[
+                  { value: 0, label: 'Monday' },
+                  { value: 1, label: 'Tuesday' },
+                  { value: 2, label: 'Wednesday' },
+                  { value: 3, label: 'Thursday' },
+                  { value: 4, label: 'Friday' },
+                  { value: 5, label: 'Saturday' },
+                  { value: 6, label: 'Sunday' },
+                ]}
+                defaultValue={config ? config.dayOfWeek : undefined}
+              />
+            </ValidationFormGroup>
+          </div>
+          <div className="col">
+            <ValidationFormGroup
+              for="hourOfDay"
+              helpText="The hour of the day to send the report, in Eastern Standard Time (EST). This is required for all frequency settings"
+              invalid={invalidFields.hourOfDay}
+              invalidMessage="Required for all frequency types"
+            >
+              <label htmlFor="hourOfDay">Hour of Day</label>
+              <Input
+                type="number"
+                id="hourOfDay"
+                name="hourOfDay"
+                defaultValue={config ? config.hourOfDay : undefined}
+                onBlur={e => this.setState({
+                  invalidFields: {
+                    ...invalidFields,
+                    hourOfDay: !e.target.value.length,
+                  },
+                })}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <ValidationFormGroup
+          for="pgpEncyptionKey"
+          helpText="The key for encyption, if PGP encrypted file is required"
+        >
+          <label htmlFor="pgpEncryptionKey">PGP Ecryption Key</label>
+          <Input
+            type="textarea"
+            id="pgpEncryptionKey"
+            name="pgpEncryptionKey"
+            defaultValue={config ? config.pgpEncryptionKey : undefined}
+          />
+        </ValidationFormGroup>
+        {deliveryMethod === 'email' && (
+          <div className="row">
+            <div className="col">
+              <ValidationFormGroup
+                for="emailRaw"
+                helpText="The email(s), one per line, where the report should be sent"
+                invalidMessage="Required. One email per line. Emails must be formatted properly (email@domain.com)"
+                invalid={invalidFields.email}
+              >
+                <label htmlFor="emailRaw">Email(s)</label>
+                <Input
+                  type="textarea"
+                  id="email"
+                  name="emailRaw"
+                  defaultValue={config ? config.email.join('\n') : undefined}
+                  onBlur={(e) => {
+                    this.setState({
+                      invalidFields: {
+                        ...invalidFields,
+                        email: !EMAIL_REGEX.test(e.target.value),
```

Comment:
There is a `isValidEmail` function in `utils.js` that you could potentially use in place of instead of the custom `EMAIL_REGEX`. The utility function makes use of the `isEmail` function from the `validator` library (i.e., `import isEmail from 'validator/lib/isEmail';`). Up to you if you want to use it :)

Reply:
This is now using that function